src/components/ArchiveTable.vue

```diff
@@ -30,7 +30,7 @@ export default {
       return this.$store.state.urls.archiveClient;
     },
     archiveLink: function() {
-      return this.archiveClientUrl + '/?request_id=' + this.requestid + '&start=2014-01-01';
+      return this.archiveClientUrl + '/?request_id=' + this.requestid + '&start=2014-01-01&public=true';
```

Comment:
Is this a good time to set reasonable bounds for `start` and `end` too?

Reply:
what were you thinking of as more reasonable bounds? The closest thing I can think of is getting all the windows, and then finding the earliest/latest window start/end as the bounds, but DIRECT submission requests won't have a window so that wouldn't work for them.

Reply:
Oh, that sounds like more work that it's worth. It doesn't have to be exact, as long as you can get it down to a year range. And it's not a critical thing either. It's just that the aggregates won't work on the archive UI. I'll leave it up to you.

Reply:
ah... since this already filters it by request id, I doubt people will use aggregates along with it since it shouldn't have very many results already.